### PR TITLE
Include linux instructions for installing MongoDB

### LINKED ISSUE #149: README don't specify how to install mongo db on linux
## Type of issue:
- [ ] Bug / Error
- [ ] Idea / Feature
- [x] Improvement detail

### PR DESCRIPTION
**Pull request by dinaelhanan**
## Changes proposed in the pull request
  * Add instructions for linux systems for installation of MongoDB
## Impact
  *Changes README.md
## Other information

Check if you have done the following before creating the PR
- [ ] console logs have been deleted
(I don't think this is necessary for this PR)
- [x ] No commented code (If commented the reasons are given)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,11 @@ For example, you can add the following line to your shell’s rc file (e.g. ~/.b
 ```export PATH=<mongodb-install-directory>/bin:$PATH```
 Replace <mongodb-install-directory> with the path to the extracted MongoDB archive.
 
+## On Linux
+For Ubuntu,SUSE, and Debian systems, you can install mongoDB Community Edition using .deb packages. The process varies from distro to distro, so I suggest you visit the link below, select your distribution.
```
Comment (bauripalash) [inline README.md]: You could have added commands to install MongoDB on those mentioned distros.

Comment (bauripalash) [reply README.md]: On Ubuntu MongoDB can be installed simply by , 
`sudo apt install mongodb`

Comment (dinaelhanan) [reply README.md]: Ok, I'll add those commands for those distros

Comment (dinaelhanan) [reply README.md]: Done! Thanks for pointing that out. I used the terminal command `sudo apt-get install -y mongodb-org` so they get the latest stable version

Comment (dinaelhanan) [reply README.md]: Done! I used `sudo apt-get install -y mongodb-org` so they get the latest stable version :)

**Comment by dinaelhanan** [comment]
@LeWarEnds @bauripalash  this has been done :) Please check it out!

**Pull request merged**